kong/plugins/aws-lambda/handler.lua

```diff
@@ -250,7 +250,13 @@ function AWSLambdaHandler:access(conf)
   -- instead of JSON arrays for empty arrays.
   if conf.empty_arrays_mode == "legacy" then
     local ct = headers["Content-Type"]
-    if ct and ct:lower():match("application/.*json") then
+    -- If Content-Type is specified by multiValueHeader then
+    -- it will be an array, so we need to get the first element
+    if type(ct) == "table" and #ct > 0 then
+      ct = ct[1]
+    end
+
+    if ct and type(ct) == "string" and ct:lower():match("application/.*json") then
```

Comment:
`application/.*json` seems not matching `application/json+test`?

Reply:
There's no need to match, I'm just deliberately setting a Content-Type value and test if it can be returned correctly.